fetch-data.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Install PostgreSQL client tools (only needed if not already installed)
+# sudo apt-get install postgresql-client
+
+# Production database credentials
+production_db_host="xxx.rds.amazonaws.com"
+production_db_name="xxx"
+production_db_user="postgres"
+production_db_password="xxx"
+
+# Local database credentials
+local_db_host="localhost"
+local_db_name="dbservice_dev"
+local_db_user="postgres"
+local_db_password="postgres"
+
```

Comment:
Let's add additional command to delete the existing tables of the local database first before creating dump in order to avoid remove irrelevant entries(if any) exisiting in the local db.